assets/app/views/editor/edit-file.js

```diff
@@ -22,55 +23,139 @@ var EditorView = Backbone.View.extend({
   },
   template: _.template(templateHtml),
   initialize: function (opts) {
-    var self      = this,
-        file      = [
-                      self.model.get('owner'),
-                      self.model.get('repoName'),
-                      self.model.get('branch'),
-                      self.model.get('file')
-                    ].join('/'),
-        raw, content, settingsEditorEl, contentEditorEl;
+    var self      = window.z = this,
+        file      = this.filePathFromModel(this.model),
+        fileExt   = this.model.get('file').split('.')[1],
```

Comment:
Good point and catch. No, it isn't. For instance, it will break on `.travis.yml` as well. 

I'll fix that up